config/200-config.yaml

```diff
@@ -58,3 +58,6 @@ data:
     # NOTE: This flag is in an alpha state and is mostly here to enable internal testing
     #       for now. Use with caution.
     cluster-cert-secret: ""
+
+    # Specifies the amount of time that Kourier waits for the incoming requests.
```

Comment:
I think that we should mention that the specified value is set to `stream_idle_timeout` in Envoy, though it is obvious from the configuration key.